src/views/settings/view_reaction_settings.cpp

```diff
@@ -95,51 +67,54 @@ namespace big
 	{
 		components::title("SETTINGS_REACTIONS"_T);
 		draw_reaction(g.reactions.bounty);
+		draw_reaction(g.reactions.break_game);
 		draw_reaction(g.reactions.ceo_kick);
 		draw_reaction(g.reactions.ceo_money);
 		draw_reaction(g.reactions.chat_spam);
-		draw_reaction(g.reactions.clear_ped_tasks);
 		draw_reaction(g.reactions.clear_wanted_level);
 		draw_reaction(g.reactions.crash);
-		draw_reaction(g.reactions.destroy_personal_vehicle);
+		draw_reaction(g.reactions.delete_vehicle);
 		draw_reaction(g.reactions.end_session_kick);
+		draw_reaction(g.reactions.destroy_personal_vehicle);
 		draw_reaction(g.reactions.fake_deposit);
 		draw_reaction(g.reactions.force_mission);
 		draw_reaction(g.reactions.force_teleport);
-		draw_reaction(g.reactions.game_anti_cheat_modder_detection);
 		draw_reaction(g.reactions.give_collectible);
 		draw_reaction(g.reactions.gta_banner);
 		draw_reaction(g.reactions.kick_from_interior);
-		draw_reaction(g.reactions.kick_vote);
 		draw_reaction(g.reactions.mc_teleport);
-		draw_reaction(g.reactions.modder_detection);
 		draw_reaction(g.reactions.network_bail);
 		draw_reaction(g.reactions.null_function_kick);
 		draw_reaction(g.reactions.personal_vehicle_destroyed);
 		draw_reaction(g.reactions.remote_off_radar);
-		draw_reaction(g.reactions.remote_ragdoll);
-		draw_reaction(g.reactions.remote_wanted_level);
-		draw_interloper_reaction(g.reactions.remote_wanted_level_others);
-		draw_reaction(g.reactions.report);
-		draw_reaction(g.reactions.report_cash_spawn);
-		draw_reaction(g.reactions.request_control_event);
 		draw_reaction(g.reactions.rotate_cam);
 		draw_reaction(g.reactions.send_to_cutscene);
-		draw_reaction(g.reactions.send_to_interior);
 		draw_reaction(g.reactions.send_to_location);
+		draw_reaction(g.reactions.send_to_interior);
 		draw_reaction(g.reactions.sound_spam);
-		draw_reaction(g.reactions.spectate);
 		draw_reaction(g.reactions.spectate_notification);
-		draw_interloper_reaction(g.reactions.spectate_others);
 		draw_reaction(g.reactions.start_activity);
 		draw_reaction(g.reactions.start_script);
 		draw_reaction(g.reactions.teleport_to_warehouse);
 		draw_reaction(g.reactions.transaction_error);
 		draw_reaction(g.reactions.trigger_business_raid);
 		draw_reaction(g.reactions.tse_freeze);
 		draw_reaction(g.reactions.tse_sender_mismatch);
-		draw_reaction(g.reactions.turn_into_beast);
 		draw_reaction(g.reactions.vehicle_kick);
+		draw_reaction(g.reactions.turn_into_beast);
+		draw_reaction(g.reactions.remote_wanted_level);
+		draw_interloper_reaction(g.reactions.remote_wanted_level_others);
+		ImGui::Separator();
```

Comment:
That's reverting some changes about the sorting made a short while ago.
Anyhow: Both sortings (old and "new" just alphabetic one) don't make overly much sense anyway.